### DISCUSSION TITLE
Spelling Mistakes

### DISCUSSION THREAD

**Discussion by Yusufknk** [IBAX-io/go-ibax#44] [Typos （English）]
There are some typos.  The first letters of the buttons can be capitalized.  The same error exists in many places.  As in "Send Coin"
![IMG_20220114_153104](https://user-images.githubusercontent.com/97254047/149515941-7fb6e0d8-5129-4414-81bf-6ecc07461514.jpg)

**Comment by HalloKyle** [comment]
Thank you so much, we will fix that shortl after our tech guy is available.
